app/models/flat.rb

```diff
@@ -1,4 +1,6 @@
 class Flat < ApplicationRecord
   belongs_to :user
 
+  validates :name, uniqueness: true
```

Comment:
Hi Michael! I think we should also validate the presence of the name in addition to its uniqueness